docs/tools/cli/service.rst

```diff
@@ -15,7 +15,7 @@ Commands for managing Aiven services via ``avn`` commands.
 
 Manages the Aiven for Apache Kafka® ACL entries.
 
-More information on ``acl-add``, ``acl-delete`` and ``acl-list`` can be found in :doc:`the dedicated page <service/acl>`.
+More information on :doc:``acl-add``, ``acl-delete`` and ``acl-list`` can be found in `the dedicated page </docs/tools/cli/service/acl.html>`.
```

Comment:
I think the :doc: is in the wrong place here, or a backtick is missing if you wanted to link that whole part of the sentence?